README.md

```diff
@@ -581,6 +583,50 @@ for other types of options.
 Many modifiers are polymorphic in this type argument, which means
 that they can be used with any builder.
 
+### Common Builder Patterns
+
+#### A list of values with a default
+
+We'll show 2 wrong ways to do it and then show the correct way to do it.
+
+Wrong way #1: using `some (strOption modifiers)`/`many (strOption modifiers)`.
+We could use `some (strOption (long "arg-name" <> value "default"))`, which allows you to pass in values like this:
+`command --arg-name value1 --arg-name value2`
+
+However, that isn't stack stafe.
+
+Wrong way #2: using `some (option) <|> NonEmptyString.singleton "default"`
+This command is stack-safe, but does not allow us to display the default value in the help text.
+
+Correct way: use `eitherReader` to define our own `ReadM` that properly handles this:
```

Comment:
You can massage "way 2" to see default value in help:

```purs
let defaultVal = NonEmptyList.singleton "a"
in some (strOption (long "ignore" <> help ("Path to ignore, default: " <> show defaultVal)) <|> defaultVal
```

So it's not really a wrong way, we can instead try to use wording like:
`We will show one incorrect way and two alternative ways of implementing it`

Reply:
I've accounted for this in the latest commits.

However, I still think the 3rd way is the best way to write this as it's consistent with single-value default values. So, I described the latter two ways as "right (but inconsistent)" and "right (and consistent)"